spark/src/test/scala-spark-master/org/apache/spark/sql/delta/DeltaVariantSuite.scala

```diff
@@ -52,9 +52,11 @@ class DeltaVariantSuite
       sql("CREATE TABLE tbl(s STRING, v VARIANT) USING DELTA")
       sql("INSERT INTO tbl (SELECT 'foo', parse_json(cast(id + 99 as string)) FROM range(1))")
       assert(spark.table("tbl").selectExpr("v::int").head == Row(99))
-      assert(
-        getProtocolForTable("tbl") ==
-        VariantTypeTableFeature.minProtocolVersion.withFeature(VariantTypeTableFeature)
+      assert(getProtocolForTable("tbl") ==
+          VariantTypeTableFeature.minProtocolVersion.withFeatures(Seq(
+            AppendOnlyTableFeature,
+            InvariantsTableFeature,
+            VariantTypeTableFeature))
```

Comment:
Do we really want to assert the exact protocol here? Or rather just that it supports `VariantTypeTableFeature`? Like these we'll have to update it every time the defaults change, no?